cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -988,6 +989,33 @@ def ceph_storage_privilege():
     reconfigure_apiserver(endpoint_from_flag('etcd.available'))
 
 
+@when('kubernetes-master.ceph.configured')
+@when_not('kubernetes-master.ceph.pool.created')
+def ceph_storage_pool():
+    '''Once Ceph relation is ready,
+    we need to add storage pools.
+
+    :return: None
+    '''
+    hookenv.log('Creating Ceph pools.')
+
+    ReplicatedPool(
+        name='xfs-pool',
+        service='admin',
+        replicas=3,
```

Comment:
should any of this be a config option? replicas? service?

Reply:
Yeah, I was going to ask.  The objection I have is that you're then repeating config options from the `ceph-mon` charm.  I guess we could change the interface so that we could pull these too?